docusaurus/docs/iOS/guides/getting-started.md

```diff
@@ -1,3 +1,308 @@
+
+#### Components respect the `tintColor` of their current view hierarchy
+
+| default `tintColor`  | `tintColor = .systemPink` |
+| ------------- | ------------- |
+| ![Chat UI with default tint color](../assets/blue-tint.png)  | ![Chat UI with pink tint color](../assets/pink-tint.png)  |
+
+<p>&nbsp;</p>
+
+#### Components support light/dark user interface style
+
+| `userInterfaceStyle = .light`  | `userInterfaceStyle = .dark` |
+| ------------- | ------------- |
+|  ![Chat UI with light user interface style](../assets/user-interface-style-light.png)  | ![Chat UI with dark user interface style](../assets/user-interface-style-dark.png)  |
+
+<p>&nbsp;</p>
+
+#### Components support dynamic content size categories
+
+| `preferredContentSizeCategory = .small`  | `preferredContentSizeCategory = .extraLarge` |
+| ------------- | ------------- |
+|  ![Chat UI with small content size category](../assets/content-size-small.png)  | ![Chat UI with extra larga content size category](../assets/content-size-extra-large.png)  |
+
+<p>&nbsp;</p>
+
+#### Custom Components can be injected into the SDK
+
+You can replace all `StreamChatUI` components with your custom subclasses using the `Components` object. It doesn't matter how deep in the hierarchy the component lives:
+
+```swift
+// Your custom subclass that changes the behavior of avatars
+class RectangleAvatarView: ChatAvatarView { 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        imageView.layer.cornerRadius = 2
+    }
+}
+
+// Register it with `UIConfig`
+Components.default.avatarView = RectangleAvatarView.self
+```
+
+| default `ChatAvatarView`  | custom `RectangleAvatarView ` |
+| ------------- | ------------- |
+|  ![Chat UI with default avatar view](../assets/default-avatars.png)  | ![Chat UI with custom rect avatar view](../assets/rect-avatars.png)  |
+
+<p>&nbsp;</p>
+
+
+## User Authentication
+
+After the SDK is installed, the next step is to initialize the chat client and authenticate the user. It can be a regular user, or it can be an [anonymous](https://getstream.io/chat/docs/anon/?language=swift) or [guest](https://getstream.io/chat/docs/guest_users/?language=swift) user with limited sets of permissions. A `TokenProvider` is the object that will hold your authentication strategy and will tell the `ChatClient` which user you're currently logged in with.
+
+## User Tokens
+
+User Tokens are JWT tokens containing a User ID and used to authenticate a user. You can [generate these tokens using a server client](https://getstream.io/chat/docs/ios-swift/tokens_and_authentication/?language=swift#generating-tokens) and then use them to authenticate a user in a chat client.
+
+:::note
+Authenticating a user with a valid user token creates the said user in Stream backend if it's not created yet.
+
+To create a new user, simply create a user token for the desired user id and connect with the said user id.
+:::
+
+### Regular User
```

Comment:
Let's have the getting started also introduce enabling developer tokens and assume their use as the default. We can provide other options further into the guide or have links to the CMS section on it the way you currently do.

Reply:
I need some clarification on this... don't we want to show other scenarios in the guide?